src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/menu/MainMenu.kt

```diff
@@ -25,7 +33,41 @@ class MainMenu : MenuBar() {
                         }
                     }
                 }
+                menu("Default Audio Plugin") {
+                    graphic = MaterialIconView(MaterialIcon.MIC, "20px")
+                    val pluginToggleGroup = ToggleGroup()
+
+                    // Get the plugins from the use case
+                    val pluginActions = PluginActions(Injector.pluginRepository)
+                    pluginActions
```

Comment:
I kind of intended for the file menu to be temporary and just needed a way to test importing resource containers. If this is meant to be anything more than very temporary, it needs model/viewmodel classes

Reply:
I was treating it as temporary as well, just to let you switch plugins and show that it worked.

Reply:
no worries!